internal/console/helpers/helpers.go

```diff
@@ -157,3 +162,39 @@ func ListReportFormats() []string {
 	sort.Strings(supportedFormats)
 	return supportedFormats
 }
+
+// GetNumCPU return the number of cpus available
+func GetNumCPU() (float32, error) {
+	// Check if application is running inside docker
+	if _, err := os.Stat("/.dockerenv"); err == nil {
+		f, err := os.Open("/sys/fs/cgroup/cpu/cpu.cfs_quota_us")
+		if err != nil {
+			return -1, err
+		}
+
+		defer func() {
+			if err := f.Close(); err != nil {
+				log.Err(err)
```

Comment:
Maybe we could be more precise in the log:

```suggestion
        log.Err(err).Msg("failed to close '/sys/fs/cgroup/cpu/cpu.cfs_quota_us'")
```